src/test/java/net/sf/jabref/logic/util/strings/StringUtilTest.java

```diff
@@ -94,20 +94,10 @@ public void testJoin() {
     public void testStripBrackets() {
         assertEquals("foo", StringUtil.stripBrackets("[foo]"));
         assertEquals("[foo]", StringUtil.stripBrackets("[[foo]]"));
-        assertEquals("foo", StringUtil.stripBrackets("foo]"));
```

Comment:
Looking at the use case for the function, it seems a bit weird that single square brackets are removed. So I removed these tests as the current implementation will not remove single brackets.